src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -4468,7 +4469,7 @@ def test_aks_migrate_cluster_to_cilium_dataplane(self, resource_group, resource_
         ])
 
         # update to enable cilium dataplane
-        update_cmd = 'aks update -g {resource_group} -n {name} --network-dataplane=cilium'
+        update_cmd = 'aks update -g {resource_group} -n {name} --network-dataplane=cilium --network-policy=cilium'
 
```

Comment:
do we need to update the test fixtures after running the live test? I would have expected this to change the PUT MC request, but don't see any changes in `recordings/test_aks_migrate_cluster_to_cilium_dataplane.yaml`

Reply:
I am not sure about test fixtures, probably @FumingZhang or @zhoxing-ms can help clarify

Reply:
Queued [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=80318220) to validate the change. If test passed, the recording file would be published as pipeline artifact, could download and commit it.

Reply:
The test failed due to 
> E           azure.core.exceptions.HttpResponseError: (PropertyChangeNotAllowed) Changing property 'networkProfile.networkPolicy' is not allowed.
> E           Code: PropertyChangeNotAllowed
> E           Message: Changing property 'networkProfile.networkPolicy' is not allowed.
> E           Target: networkProfile.networkPolicy